DOC-2689: Documentation enhancements for `autoresize` plugin in `v8` docs. (#3831)
tinymce/tinymce-docs: modules/ROOT/pages/autoresize.adoc

:navtitle: Autoresize
:description_short:
:description: Automatically resize TinyMCE to fit content.
:keywords: height, width, max_height, min_height, autoresize_overflow_padding, autoresize_bottom_margin, resize
:pluginname: Autoresize
:plugincode: autoresize

> The Autoresize plugin automatically adjusts the editor height to match its content, preventing the editor from expanding infinitely as users type. This creates a more intuitive editing experience where the editor grows and shrinks naturally with content.

Reviewer: ```suggestion
The {pluginname} plugin automatically adjusts the editor height to match its content. This creates a more intuitive editing experience where the editor grows and shrinks naturally with content. Setting the `+max_height+` option would allow you to still control when the content would become scrollable.
```
Without referring to `max_height` this plugin would allow the content to expand infinitely, so that part no longer makes sense.

});
----

== How it works

The Autoresize plugin monitors content changes and automatically adjusts the editor height to accommodate the content. When enabled:

* The editor initializes and renders based on the minimum height specified by `+min_height+` option.
* As content is added, the editor expands vertically
* The editor stops expanding when it reaches the `+max_height+` limit

> * The `+resize+` handle is automatically disabled to prevent manual resizing conflicts

Reviewer: ```suggestion
* The `+resize+` handle is removed from the editor.
```
